src/codetypes.h

```diff
@@ -55,8 +57,31 @@
 #define HOLEMAPSIZE        192
```

Comment:
This number is the number of auxiliary mesh elements in each director of the uniform Cartesian hole map size (192 x 192 x 192). For every mesh body, a hole map is that size is constructed (with each cell marked as a hole or outer value). Originally, Jay had HOLEMAPSIZE as 64, which is actually too small because you would likely have nodes marked as wall-bc and outer-bc falling into the same Cartesian element (which is prevents the correct identification of hole cells).
The number 192 was set by your team (Shreyas) -- it's a nice compromise between taking tons of memory and the computational speed to traverse the hole Cartesian map (called a SAM-Structured Auxiliary Mesh). The most you could really go (depending on how many cores and RAM/core) is ~768 -- but the wall-clock time explodes along with the memory.